### ISSUE TITLE
Example of accessing SuperCDMS data on OSN

### ISSUE THREAD

**Issue by zonca** [pondd-project/pondd#12]
As part of developing a DID finder for data on OSN #4, we want to have an example.

https://gitlab.com/supercdms/DataHandling/OSNTransfer/-/blob/master/examples/AccessToOSN.py

**Comment by zonca** [comment]
Thanks for providing this script.

* why is the data in `joblib` format? isn't it usually in root files?
* it seems that is a power spectrum, any plan to store raw data (I mean timelines)?
* it seems you have an example file. Do you have any complete dataset staged on OSN?

**Comment by pibion** [comment]
@zonca this particular script is for a dataset that was taken with a rogue DAQ.  We do store raw data for CDMS, would an example accessing raw data for CDMS be useful?

We have some complete datasets staged on OSN, but we don't have metadata stored for all of them - I suggested we start with just one, but it does seem like some metadata on a data set would make sense!

I'll work with Haythem and see if we can find a good set of data to tag.

**Comment by zonca** [comment]
> @zonca this particular script is for a dataset that was taken with a rogue DAQ. We do store raw data for CDMS, would an example accessing raw data for CDMS be useful?

@pibion for "raw" I meant the timelines, like the ones in the example tutorial: https://gist.github.com/zonca/62784b06fabd72b254c2a7f7252eeafc

My assumption was that the OSN DID finder will host the same data as the SuperCDMS Catalog DID finder just with a different storage backend. Is this correct? Or do you plan to have different kind of data in OSN?

> We have some complete datasets staged on OSN, but we don't have metadata stored for all of them - I suggested we start with just one, but it does seem like some metadata on a data set would make sense!
> 
> I'll work with Haythem and see if we can find a good set of data to tag.

very good

**Comment by zonca** [comment]
@pibion any updates about SuperCDMS data on OSN?

**Comment by pibion** [comment]
@zonca yes, I have a student who's working on adding metadata to a dataset we're currently analyzing.  He's got a lot of classes so this may be a bit slow, but we're moving!

**Comment by zonca** [comment]
@pibion is this still relevant, or should I close the issue?
